content/repositories/working-with-files/using-files/index.md

```diff
@@ -8,7 +8,7 @@ versions:
   ghec: '*'
 children:
   - /navigating-code-on-github
-  - /tracking-changes-in-a-file
+  - /viewing-a-file
```

Comment:
Since we changed the name of the file, we also need to update the index file that has the article ⚡